### ISSUE TITLE
Unable to reproduce result on COCO2014

### ISSUE THREAD

**Issue by marzi-heidari** [computationalmedia/semstyle#7]
I'm want to reproduce the result of captioning for validation set of COCO 2014, but the captions I've got are different from the ones in [https://almath123.github.io/semstyle_examples/](https://almath123.github.io/semstyle_examples/).
for instance for the image with id 74 the caption that I'm getting is "i 'm a dog sitting on the bike"  but it must be
"The dog was sitting on the ground."
 I wonder if there is something that I'm missing? Is there any necessary pre-processing?

**Comment by almath123** [comment]
There are slight differences between the original theano implementation (in the original_code folder) and the pytorch re-implementation. You should be able to get exactly the captions in the examples only if you use the theano implementation with the pre-trained models (since training is somewhat random). The theano implementation is challenging to setup because of the dependencies hence the re-implementation.

**Comment by marzi-heidari** [comment]
Were these differences put there intentionally (for performance or something) or because of differences in theano and pytorch you have no choice but considering some differences in re-implementation?

**Comment by marzi-heidari** [comment]
I really am sorry for asking too many questions but I've got another one:
I want to use theano implementation for reproducing result on COCO2014 but in addition to issue #8 I need a file named img_to_tags_framenet_02_11_1719_e.pik for doing so.
Can you please add files that are necessary for testing on theano implementation?

**Comment by almath123** [comment]
The differences in going from the old code to the new code were to improve computational efficiency, portability, and readability. I also removed large parts of the code that were effectively dead (not used but still in the original source).

The img_to_tags_framenet_02_11_1719_e.pik file should be available by running the download.sh in /semstyle/original_code/models .
